hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionSplitPolicy.java

```diff
@@ -75,7 +75,7 @@ protected void configureForRegion(HRegion region) {
    */
   protected boolean canSplit() {
     return !region.getRegionInfo().isMetaRegion() && region.isAvailable() &&
-      !region.hasReferences();
+      region.getStores().stream().allMatch(HStore::canSplit);
```

Comment:
This is an interesting change. Is behavior still the same?

Reply:
Yeah, I think they are equivalent.
Thanks.